create_kubernetes_sysext.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -euo pipefail
+
+export ARCH="${ARCH-amd64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME CNI_VERSION"
+  echo "The script will download the Kubernetes release binaries (e.g., for v1.27.3) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  echo "CNI version current value is 'latest'"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+CNI_VERSION="${3-latest}"
+
```

Comment:
```suggestion
if [ "${ARCH}" = aarch64 ]; then
  ARCH=arm64
fi
```
Sometimes arm64 vs aarch64 is confusing, let's help the user a bit